superset-frontend/packages/superset-ui-core/src/query/types/Datasource.ts

```diff
@@ -53,7 +53,4 @@ export const DEFAULT_METRICS: Metric[] = [
   },
 ];
 
-export const isValidDatasourceType = (datasource: DatasourceType) =>
```

Comment:
this function is unused. a more readable expression is `'Table' in DatasourceType`.